database/factories/UserFactory.php

```diff
@@ -6,41 +6,49 @@
 use Illuminate\Database\Eloquent\Factories\Factory;
 use Illuminate\Support\Str;
 
-class UserFactory extends Factory
-{
-    /**
-     * The name of the factory's corresponding model.
-     *
-     * @var string
-     */
+class UserFactory extends Factory {
+
     protected $model = User::class;
 
-    /**
-     * Define the model's default state.
-     *
-     * @return array
-     */
-    public function definition()
-    {
+    public function definition() {
         return [
-            'name' => $this->faker->name(),
+            'name' => $this->faker->firstName(),
+            'surname' => $this->faker->lastName(),
+            'account_name' => $this->faker->unique()->colorName(),
             'email' => $this->faker->unique()->safeEmail(),
+            'role_id' => 3,
             'email_verified_at' => now(),
+            'is_active' => true,
             'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
             'remember_token' => Str::random(10),
         ];
     }
 
-    /**
-     * Indicate that the model's email address should be unverified.
-     *
-     * @return \Illuminate\Database\Eloquent\Factories\Factory
-     */
-    public function unverified()
-    {
+    public function unverified() {
         return $this->state(function (array $attributes) {
             return [
                 'email_verified_at' => null,
+                'is_active' => false
+            ];
+        });
+    }
+
+    public function admin() {
+        return $this->state(function (array $attributes) {
+            return [
+                'account_name' => 'admin',
+                'password' => 'admin',
+                'role_id' => 1,
+            ];
+        });
+    }
+
+    public function moderator() {
+        return $this->state(function (array $attributes) {
+            return [
+                'account_name' => 'mod',
+                'password' => 'mod',
```

Comment:
w seedach proponuję dla wszystkich kont ustawiać hasło jako password + tutaj dajesz password jako mod co oznacza że w bazie też będzie mod i to trochę słabo bo podasz mod do logowania to zrobi z mod podanego przez usera hasha i porówna do stringa mod. wydaje mi się że tutaj powinno być Hash::make('mod');

Reply:
Dam hasło "admin" to i tak do testów tylko, a krócej się piszę niż password 🐌. Co do haszowania to jasne, w sumie od razu powinienem dać, ale stwierdziłem, że do logowania/rejestracji doda się ten feature. Ale rzeczywiście lepiej od razu.